src/webui/www/private/statistics.html

```diff
@@ -1,59 +1,113 @@
-    </tr>
-    <tr>
-        <td>QBT_TR(All-time download:)QBT_TR[CONTEXT=StatsDialog]</td>
-        <td id="AlltimeDL" class="statisticsValue"></td>
-    </tr>
-    <tr>
-        <td>QBT_TR(All-time share ratio:)QBT_TR[CONTEXT=StatsDialog]</td>
-        <td id="GlobalRatio" class="statisticsValue"></td>
-    </tr>
-    <tr>
-        <td>QBT_TR(Session waste:)QBT_TR[CONTEXT=StatsDialog]</td>
-        <td id="TotalWastedSession" class="statisticsValue"></td>
-    </tr>
-    <tr>
-        <td>QBT_TR(Connected peers:)QBT_TR[CONTEXT=StatsDialog]</td>
-        <td id="TotalPeerConnections" class="statisticsValue"></td>
-    </tr>
-</table>
-
-<h3>QBT_TR(Cache statistics)QBT_TR[CONTEXT=StatsDialog]</h3>
-<table style="width:100%">
-    <tr>
-        <td>QBT_TR(Read cache hits:)QBT_TR[CONTEXT=StatsDialog]</td>
-        <td id="ReadCacheHits" class="statisticsValue"></td>
-    </tr>
-    <tr>
-        <td>QBT_TR(Total buffer size:)QBT_TR[CONTEXT=StatsDialog]</td>
-        <td id="TotalBuffersSize" class="statisticsValue"></td>
-    </tr>
-</table>
-
-<h3>QBT_TR(Performance statistics)QBT_TR[CONTEXT=StatsDialog]</h3>
-<table style="width:100%">
-    <tr>
-        <td>QBT_TR(Write cache overload:)QBT_TR[CONTEXT=StatsDialog]</td>
-        <td id="WriteCacheOverload" class="statisticsValue"></td>
-    </tr>
-    <tr>
-        <td>QBT_TR(Read cache overload:)QBT_TR[CONTEXT=StatsDialog]</td>
-        <td id="ReadCacheOverload" class="statisticsValue"></td>
-    </tr>
-    <tr>
-        <td>QBT_TR(Queued I/O jobs:)QBT_TR[CONTEXT=StatsDialog]</td>
-        <td id="QueuedIOJobs" class="statisticsValue"></td>
-    </tr>
-    <tr>
-        <td>QBT_TR(Average time in queue:)QBT_TR[CONTEXT=StatsDialog]</td>
-        <td id="AverageTimeInQueue" class="statisticsValue"></td>
-    </tr>
-    <tr>
-        <td>QBT_TR(Total queued size:)QBT_TR[CONTEXT=StatsDialog]</td>
-        <td id="TotalQueuedSize" class="statisticsValue"></td>
-    </tr>
-</table>
+<!DOCTYPE html>
+<html lang="${LANG}">
+
+<head>
+    <meta charset="UTF-8" />
+    <title>QBT_TR(Statistics)QBT_TR[CONTEXT=StatsDialog]</title>
+    <link rel="stylesheet" href="css/style.css" type="text/css" />
+    <script src="scripts/lib/mootools-1.2-core-yc.js"></script>
+    <script src="scripts/lib/mootools-1.2-more.js"></script>
+    <script src="scripts/misc.js"></script>
+    <script>
+        var loadStatisticsTimer;
+
+        window.addEvent('domready', function() {
+            getStats();
+        });
+
+        function getStats() {
+            new Request.JSON({
+                url: 'api/v2/sync/stats',
+                noCache: true,
+                method: 'get',
+                onFailure: function() {
+                    $('error_div').set('html', 'QBT_TR(qBittorrent client is not reachable)QBT_TR[CONTEXT=HttpServer]');
+
+                    clearTimeout(loadStatisticsTimer);
+                    loadStatisticsTimer = getStats.delay(5000);
```

Comment:
is it required to call `clearTimeout()`? can't you just write `getStats.delay(5000);` without the above 2 lines?

Reply:
Calling `delay()` doesn't clear the previous timer, so it will still go off.